frontend/app/controllers/home.js

```diff
@@ -1,4 +1,48 @@
-import Controller from '@ember/controller';
-
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
 export default Controller.extend({
+  router: service("-routing"),
+  _restaurantService: service("restaurant-service"),
+  showAutoComplete: false,
+  selectedRestaurant: false,
+  query: "",
+  selectedRestaurantId: null,
+  suggestions: [],
+
+  search() {
+    let q = this.get("query");
+    console.log(q);
+    if (q != "") {
+      this.get("_restaurantService")
+        .filter("q", q, 1, 5)
+        .then(response => {
+          console.log("SELECT:");
```

Comment:
remove console logs

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb